songsuro/autoencoder/encoder/encoder.py

```diff
@@ -0,0 +1,114 @@
+import torch
+from torch import nn
+from .vconv import VirtualConv, output_offsets
+from .netmisc import xavier_init
```

Comment:
Recommend to not use relative path import
It can be occur some issues when users install it via pypi packages.